### PR TITLE
feat(client): design category page

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [X] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [X] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by nihalanisumit** [comment]
Screenshot from 2019-08-27 14-36-15
Screenshot from 2019-08-27 14-36-09
Category-final

File: client/src/App.jsx [hunk 1]
```diff
@@ -7,11 +7,12 @@ import 'semantic-ui-css/semantic.min.css';
 import Layout from './components/hoc/Layout/Layout';
 import Home from './containers/Home/Home';
 import BlogBox from './components/blogBox';
-import Authentication from './containers/Authentication';
+import Authentication from './containers/Authentication/Authentication';
```
Comment (piyushagwl007) [inline client/src/App.jsx:10]: Authentication file should be index.jsx

File: client/src/App.jsx [hunk 2]
```diff
@@ -7,11 +7,12 @@ import 'semantic-ui-css/semantic.min.css';
...
 import * as actions from './store/actions/index';
 import Loader from './components/UI/Loader';
-import Category from "./containers/Category";
-import CategoryType from "./containers/CategoryType";
+import Category from "./containers/Category/Category";
```
Comment (piyushagwl007) [inline client/src/App.jsx:13]: sam here

**Review by piyushagwl007 — APPROVED** [review]
LGTM after chnages

File: client/src/App.jsx (hunk 2 above)
Comment (nihalanisumit) [reply client/src/App.jsx:13]: will do refactoring in last on all the components after done developing. Rasining an issue for this.

**Pull request merged**